test/db/cmd/write

```diff
@@ -300,7 +300,6 @@ NAME=wci should commit the changes to the file
 FILE=malloc://1024
 BROKEN=1
 CMDS=<<EOF
-!rm -rf .tmp
```

Comment:
why this? what if for some reasons `.tmp` already exists?

Reply:
same reason as https://github.com/rizinorg/rizin/pull/2818#discussion_r924193289